requests-for-comments/2024-06-21-warehouse-launch-plan.md

```diff
@@ -0,0 +1,50 @@
+## [Marketing] Data warehouse launch plan
+
+The data warehouse has been in an opt-in beta for a while, and has now been opened up to a public open beta. It'll be at least a few weeks before it launches (we need to give current users two weeks notice of pricing changes) BUT that means it's the perfect time to plan. 
+
+### Ahead of launch
+
+- [ ] Publish a Stripe data warehouse tutorial - Marketing
+- [ ] Publish a Hubspot data warehouse tutorial - Marketing
+- [ ] Publish a Postgres data warehouse tutorial - Marketing
+- [ ] Update docs as needed - Marketing
+- [ ] Add Data Warehouse to in-app 'Explore the docs' menu - @joethreepwood
+- [x] Key art - @lottiecoxon (we have some already, open to new though)
+
+
+### Launch day activity
+
+- [ ] Publish a launch blog piece about why data warehouse is important - 
+- [ ] Update pricing page with data warehouse pricing
+- [ ] Update in-app onboarding with data warehouse option
+- [ ] Share blogpost to HackerNews - Marketing
```

Comment:
Do we want to do this as a launch HN type thing @jamesefhawkins or are we just going to get destroyed here because it's super early.